Userland/Applets/Keymap/main.cpp

```diff
@@ -18,11 +19,15 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
     auto app = TRY(GUI::Application::try_create(arguments));
 
     auto window = TRY(KeymapStatusWindow::try_create());
+    KeymapWindowManager wm(window);
     window->set_has_alpha_channel(true);
     window->set_title("Keymap");
     window->resize(16, 16);
     window->show();
-    window->make_window_manager(WindowServer::WMEventMask::KeymapChanged);
+
+    GUI::ConnectionToWindowManagerServer::the().async_set_event_mask(
+        WindowServer::WMEventMask::KeymapChanged);
+    GUI::ConnectionToWindowManagerServer::the().async_set_window_manager(wm.wm_id(), true);
```

Comment:
Again, this refactor should be done before doing the WindowManager split. The rest of the changes should then be squashed into the split itself.